### ISSUE TITLE
Replace .aprc with .amazing_print.yml

### ISSUE THREAD

**Issue by HarlemSquirrel** [amazing-print/amazing_print#63]
To avoid conflicts like #54 I think we should stop using `.aprc` and use a new file in YAML format like [Rubocop does](https://docs.rubocop.org/rubocop/1.1/configuration.html). 

We could support loading files from multiple places. See https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/config_loader.rb

This is also more safe since today any Ruby code in `.aprc` is executed.

**Comment by kikonen** [comment]
yes, executable code in .aprc can possibly allow code injection attacks (one reason why I never kept awesome_print enabled in production. Another reason being that it was was triggering severe performance issues in some situations, by simply having gem loaded into process; don't know if amazing_print has inherited those performance issues).

**Comment by klausbadelt** [comment]
The current fixed `.aprc` location in the *home* folder - as opposed to project folder etc.  - also complicates deployment and configuration sharing between devs. For example, building a Docker image with files outside the project folder is not possible/inconvenient.

**Comment by stefanwascoding** [comment]
+1 for using Rubocop-style config loading.

I was about to open an issue for supporting an XDG base directory aware config path (`$XDG_CONFIG_HOME/aprc` -- I like a clean $HOME 😄), when I came across this issue here, which potentially already includes that feature.

@paddor OTOH if work on this issue is not likely any time soon, let me know if a PR for just adding XDG base dir support would have a chance of getting accepted.

**Comment by paddor** [comment]
I don't have a strong opinion about this. XDG seems like a good idea. PR welcome.
